src/App.tsx

```diff
@@ -380,6 +430,14 @@ export default class App extends React.Component<{}, { expandFolderBrowser: bool
     await this.loginComponent.current?.refreshTokenAsync(true);
   }
 
+  setShouldShowSelect(): boolean {
```

Comment:
setShouldShowSelect is not a great name, makes it seem like it's setting something

just shouldShowSelect or getShouldShowSelect

same for Open, although maybe just remove that until we implement it